src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const actionsArray = [{ ...state }];
+
+  for (const action of actions) {
+    const stateCopy = { ...actionsArray[actionsArray.length - 1] };
```

Comment:
you can create this var before the loop
```suggestion
    const stateCopy = {...state};
```